docs/source/supported_models.md

```diff
@@ -28,6 +28,11 @@ AutoModelForCausalLM.from_pretrained(<model>, device_map="auto")`
 AutoModelForSeq2SeqLM.from_pretrained(<model>, device_map="auto")
 ```
 
+If you wish to serve a supported model that already exists on a local folder, you can use `weight-cache-override` flag like below. Otherwise, it will be downloaded to Hugging Face Hub cache.
```

Comment:
```suggestion
If you wish to serve a supported model that already exists on a local folder, Just point to the local folder.

text-generation-launcher --model-id <PATH-TO-LOCAL-BLOOM>
```

This is a bit simpler no ?

Reply:
I think it's better to show the opposite of what will happen if they don't do that given this is against what's happening by default. WDYT?

Reply:
I don't understand.

Default use case: -> Point to hub `model_id`.
Local model -> use the folder as `model_id`.

`weight-cache-override` does exist but it doesn't seem to be the easiest way to point users to.
For docker you can use `-v $volume:/data` which basically already cover the case, and in case of CLI and classifc `huggingface_hub` override will work (there are numerous env flags for it)

Reply:
> Default use case: -> Point to hub `model_id.`
Local model -> use the folder as `model_id`.

this is what I meant as well, but if you think my point doesn't go across this way I can commit your suggestion.